### PR TITLE
Removed extraneous web methods

### PR DESCRIPTION
**Pull request by jbrown-ultrabit**

### CODE REVIEW

File: IdleServer.js [hunk 1]
```diff
@@ -52,11 +46,11 @@ const SERVER_CONFIG = {
 const PLAYFAB_ENVIRONMENT_CONFIG = {
 	configs: {
 		"18A45": {
-			apiKey: "C5NQN8YPPCOF43HSJEWOJNZ9Z7FI9IMSIZDGU7H1CZEMJTNZT6", // playfab auto
+			apiKey: "ORX7Y8EKSK5DD1MOKUE86X5D64RQ774359ICEJJMN7U176GOBF", // playfab auto
```
Comment (LuisUltrabit) [inline IdleServer.js:49]: I think we do not need this either, at least until we have to hit outside the server sdk or cloud script helper objects

File: IdleServer.js [hunk 2]
```diff
@@ -52,11 +46,11 @@ const SERVER_CONFIG = {
...
 			logglyTag: 'idle-spongebob-auto-ops'
 		},
 		"3D16B": {
-			apiKey: "1N6S3FUPY9UZPCPRHA5J1UR7TSM3UCA3AERMCIOZ6TFGD3693B",   // playfab kongregate
+			apiKey: "9QPQ1B5Q64G8Q3DAJE9WUFTC957UP3MZADYICBD97SWBXG853J",   // playfab kongregate
```
Comment (LuisUltrabit) [inline IdleServer.js:53]: I think we do not need this either, at least until we have to hit outside the server sdk or cloud script helper objects

File: IdleServer.js [hunk 3]
```diff
@@ -1216,20 +1060,6 @@ CloudRequest.prototype.GetPlayfabServerApiKey = function () {
...
-	var rawResponse = http.request('http://www.kongregate.com/api/authenticate.json?user_id=' + kongId + '&game_auth_token=' + kongToken + '&api_key=' + apiKey);
-	var json = JSON.parse(rawResponse);
-	return json.success;
-}
-
-CloudRequest.prototype.IsWeb = function () {
-	return (SERVER_CONFIG.webGameIds.indexOf(this.GetGameId()) >= 0);
-}
-
 CloudRequest.prototype.IsPlayfab = function () {
```
Comment (LuisUltrabit) [inline IdleServer.js:1063]: we should be able to remove all `CloudRequest` as we do not use it anymore

File: IdleServer.js [hunk 4]
```diff
@@ -40,21 +40,12 @@ const SERVER_CONFIG = {
...
-		"3D16B": {
-			apiKey: "9QPQ1B5Q64G8Q3DAJE9WUFTC957UP3MZADYICBD97SWBXG853J",   // playfab kongregate
-			logglyTag: 'idle-spongebob-kongregate-ops'
-		}
 	},
-	playfabIds: ["3D16B", "18A45"]
+	logglyTag: {
+                "3D16B": "idle-spongebob-kongregate-ops", // kongregate/production
+                "19081": "idle-spongebob-auto-ops", // staging
+                "18A45": "idle-spongebob-auto-ops"  // auto
```
Comment (LuisUltrabit) [inline IdleServer.js:47]: I believe you will also need to create the respective tags in Loggly

Comment (jbrown-ultrabit) [reply IdleServer.js:47]: Already done!  :D

File: IdleConstants.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const LOGGLY_TOKEN = '00cced27-e9fd-49e2-b332-87d611e71937';
+const DEFAULT_KONG_NAME = "NoKongName";
+
+const DEFAULT_GUILD_ID = "NoId";
+const DEFAULT_GUILD_TYPE = "NoType";
+const DEFAULT_GUILD_NAME = "NoName";
+
+const DEFAULT_GUILD_COMBINED_ID = DEFAULT_GUILD_ID + "|" + DEFAULT_GUILD_TYPE + "|" + DEFAULT_GUILD_NAME;
```
Comment (LuisUltrabit) [inline IdleConstants.js:29]: I think we should be safe if we remove these

File: IdleConstants.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        },
+        logglyTag: {
+                "3D16B": "idle-spongebob-kongregate-ops", // kongregate/production
+                "19081": "idle-spongebob-auto-ops", // staging
+                "18A45": "idle-spongebob-auto-ops"  // auto
+        }
+};
+
+const LOGGLY_TOKEN = '00cced27-e9fd-49e2-b332-87d611e71937';
+const DEFAULT_KONG_NAME = "NoKongName";
```
Comment (LuisUltrabit) [inline IdleConstants.js:23]: we should be save if we move `DEFAULT_KONG_NAME` to `IdleServerPlayerProfile.js`

**Pull request merged**